pubsub/gossipsub/README.md

```diff
@@ -298,26 +305,26 @@ for each topic in mesh:
      emit PRUNE(topic) control message to peer
 ```
 
-The `fanout` map is maintained by keeping track of last published time
-for each topic:
+The `fanout` map is maintained by keeping track of the last published time a 
+topic for each topic:
```

Comment:
this a typo? the sentence doesn't make sense. Can we just revert it and not change it?

Reply:
I meant to just add "the". I'm on the phone now, will edit this and the other stuff tomorrow.